op-acceptance-tests/tests/ecotone/fees_test.go

```diff
@@ -17,6 +17,9 @@ func TestFees(gt *testing.T) {
 	require := t.Require()
 
 	require.True(sys.L2Chain.IsForkActive(forks.Ecotone), "Ecotone fork must be active for this test")
+	if sys.L2Chain.IsForkActive(forks.Isthmus) {
+		t.Skip("skipping since an Isthmus network may have an incompatible fee calculation")
```

Comment:
This will still [fail the test](https://github.com/ethereum-optimism/optimism/blob/f9c6bf2a53155ed02e326916c923eb19f0705388/op-devstack/devtest/testing.go#L242) if preconditions are expected to be met. We could use the go test object `gt` instead, but I don't think that addresses the root of the problem that op-acceptor should probably use composition instead of inheritance for gates (personally I would favor making gates manually-curated lists of tests).

It's easy enough to read the logs and see what happens, so I think keeping with convention and using `t.Skip` is worth it here.